conf/config.yaml.example

```diff
@@ -165,6 +165,7 @@ nginx_config:                     # Config for render the template to generate n
   meta:
     lua_shared_dict:              # Nginx Lua shared memory zone. Size units are m or k.
       prometheus-metrics: 15m
+      standalone-config: 10m
```

Comment:
Does it support stream_route? It cannot be used in the stream subsystem.

Reply:
I don't know about this, this PR is just providing a new data source for the config_yaml module, other than that, it's reusing previous code.
If the user is just starting a pure stream subsystem, then the Admin API is not possible to turn on, that's a server block in `http {}`, but turning on both subsystems at the same time and updating the stream route configuration via the Admin API should be allowed.

The standalone not being supported in the stream subsystem is not expected, and if it is indeed not allowed it must be fixed in a future PR. Some Kubernetes scenarios require it to work correctly.

Reply:
> but turning on both subsystems at the same time and updating the stream route configuration via the Admin API should be allowed.

Because different subsystems cannot use the same shared memory. I think you can add some test cases for this.

Reply:
https://github.com/api7/apisix-nginx-module/pull/56

I'm using exactly the same extra commands that can coexist in both subsystems at the same time, as in our prometheus shared dict.
According to tests of that PR, it allows accessing the same shared memory area in different subsystems.

Reply:
Got it.